less/defaults.less

```diff
@@ -263,12 +264,21 @@ textarea {
 }
 
 input[type="text"],
+input[type="number"],
 input[type="search"],
 input[type="password"],
 textarea {
   border: 1px solid @light-gray;
   box-shadow: inset 1px 1px 4px 0 @light-gray;
   width: 100%;
+  &.error {
```

Comment:
I think that's a cool idea, but I think we should adopt it all at once, not piecemeal. We've got a bunch of other class names (`clickable`, `hidden`, `visuallyhidden`) that don't follow that guideline, and it's a lot nicer to have one style of names. Sound good?